admin/attributes_controller.php

```diff
@@ -699,10 +698,14 @@ function popupWindow(url) {
   window.open(url,'popupWindow','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=no,resizable=yes,copyhistory=no,width=600,height=460,screenX=150,screenY=150,top=150,left=150')
 }
 </script>
+<?php
```

Comment:
This puts the above JavaScript outside the html tags

Reply:
Update your branch and do a view source - it looks ok to me.

Reply:
Is see the same errors in some of the files in #1935 where `require_once('includes/template/common/tplHtmlHeadLegacy.php');`
is put after the javascripts